src/Model/Restriction.php

```diff
@@ -293,8 +293,9 @@ public function apply() {
                 // send restriction mails to all affected bookings
                 $this->sendRestrictionMails( $bookings );
 
+				$cancelPreventionSetting = \CommonsBooking\Settings\Settings::getOption( 'commonsbooking_options_restrictions', 'restrictions-no-cancel-on-total-breakdown' ) == 'on';
```

Comment:
@hansmorb hier würde ich `$userEnabledBookingCancellationOnTotalBreakdown` schreiben (o.Ä.), dann sparst du dir das negieren der Variable und es wird mMn. lesbarer. Ansonsten muss ich ja erst verstehen was in den Settings gesetzt wird, also die Semantik des Werts der Variable.

Reply:
Das stimmt, das Einstellungsfeld an sich heißt aber restrictions-no-cancel-on-total-breakdown . Also nur, wenn es aktiviert ist, werden keine Restrictions storniert. Das hat den Hintergrund, dass es relativ kompliziert ist in CMB2 ein "default on" Feld zu machen (und dann auch eine Migrationsfunktion geschrieben werden müsste, die das automatisch anschaltet). Damit haben wir uns in der Vergangenheit schonmal etwas in den Fuß geschossen, da sind dann beim Standort keine E-Mails mehr angekommen weil die E-Mail an den Standort nur versendet wurde, wenn ein always on Häkchen gesetzt war.

Reply:
Was ist mit `$userDisabledBookingCancellationOnTotalBreakdown` und dann die Negation behalten?